website/client/components/static/staticWrapper.vue

```diff
@@ -0,0 +1,88 @@
+<template lang='pug'>
+  div
+    static-header
+
+    app-page(v-if='$route.name === "app"')
```

Comment:
instead of adding all these `v-if`s you should use something like this https://github.com/HabitRPG/habitica/blob/develop/website/client/router.js#L107-L115, then the wrapper will just need to contain a `router-view` element like here https://github.com/HabitRPG/habitica/blob/develop/website/client/components/inventory/index.vue#L8. The only thing is that this way /home will have to be a children of /static so it'll become /static/home and the link the in the static header will have to be changed